src/transformers/models/deta/configuration_deta.py

```diff
@@ -46,6 +46,9 @@ class DetaConfig(PretrainedConfig):
             is `False`, this loads the backbone's config and uses that to initialize the backbone with random weights.
         use_pretrained_backbone (`bool`, *optional*, `False`):
             Whether to use pretrained weights for the backbone.
+        use_timm_backbone (`bool`, *optional*, `False`):
```

Comment:
We need to have the `use_timm_backbone` included in all model configs that contain backbone information to differentiate between loading a transformers or timm pretrained backbone.